bin/run_tests

```diff
@@ -3,4 +3,5 @@ bin=`dirname "${BASH_SOURCE-$0}"`
 bin=`cd "$bin">/dev/null; pwd`
 source conf/drillTestConfig.properties
 cd ${bin}/../
+#java -cp conf/:${JDBC_DRIVER_CP}:framework/target/framework-1.0.0-SNAPSHOT-shaded.jar:${HADOOP_INSTALL_LOC}/lib/* org.apache.drill.test.framework.TestDriver $*
```

Comment:
Can we add a comment to each line, indicating which is for using the Apache driver and which is for using Simba driver?